Gui/opensim/view/src/org/opensim/threejs/ModelVisualizationJson.java

```diff
@@ -1689,11 +1689,12 @@ else if (secondIndex == -1){ // Conditional Path point that's inactive
                 pathpointActive_jsonArr.add(false);
                 pathpoint_jsonArr.add(pathpoint_uuid.toString());
                 // traverse backward to an active point
+                /*
```

Comment:
If this is no longer needed, would be better just to delete this section. If this is helpful to keep around, it would be good to add a comment about what this code was attempting to solve and why this didn't work.

Reply:
Deleted, thanks @carmichaelong